src/Header.tsx

```diff
@@ -58,11 +55,22 @@ const Header = () => {
         </nav>
       </div>
       <div>
-        <Link to="/login">
-          <button type="button" className="login-btn">
-            로그인
-          </button>
-        </Link>
+        {!accessToken ? (
+          <Link to="/login">
+            <button type="button" className="login-btn">
+              로그인
+            </button>
+          </Link>
+        ) : (
+          <Link to="/mypage">
+            <img
+              src={mypage_ic}
+              alt="마이페이지아이콘"
+              width="40"
+              height="40"
+            />
+          </Link>
+        )}
```

Comment:
현재는 아래와 같은 구조 인데요.
```
!accessToken ? (마이페이지 컴포넌트) : (로그인컴포넌트)
```

삼항연산자의 조건부를 바꾸면 가독성에 도움이 될 것 같아요.
```
accessToken ? (로그인컴포넌트) : (마이페이지 컴포넌트)
```

```suggestion
{!accessToken ? (
          <Link to="/mypage">
            <img
              src={mypage_ic}
              alt="마이페이지아이콘"
              width="40"
              height="40"
            />
          </Link>
        )  : ( <Link to="/login">
            <button type="button" className="login-btn">
              로그인
            </button>
          </Link>
        )}
```